test/HeaderFooter/Footer.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import Footer from '../../src/components/Footer/Footer';
+
+describe('Check Component creation', () => {
+    const element = shallow(<Footer/>);
+    test('Footer component snapshot test', () => {
+        expect(element).toMatchSnapshot();
+    });
+    test('Footer component 1', () => {

```

Comment:
This could be better explained what the test should do

Reply:
Okay